libdnf/module/ModulePackageContainer.hpp

```diff
@@ -85,6 +98,9 @@ struct ModulePackageContainer
 private:
     class Impl;
     std::unique_ptr<Impl> pImpl;
+
+    class Persistor;
```

Comment:
Why you did not create a Persistor inside pImpl instance?

Reply:
I can move it if you think it's better. I just thought that separating the persistor from the pImpl was better.

Reply:
I don't see any benefit